### PR TITLE
Added support for JWT tokens in all the required methods.

### PR DESCRIPTION
**Pull request by rtorres87**

### CODE REVIEW

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java [hunk 1]
```diff
@@ -49,12 +50,11 @@ private AppManagementService() {
     public static AppAccessToken requestAppToken(AppTokenRequest request) throws PlenigoException {
         Map<String, String> body = new LinkedHashMap<String, String>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java [hunk 2]
```diff
@@ -84,10 +84,10 @@ private static AppAccessToken buildAppAccessToken(Map<String, Object> response)
         Map<String, Object> params = new LinkedHashMap<String, Object>();
         params.put(ApiParams.CUSTOMER_ID, request.getCustomerId());
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java [hunk 3]
```diff
@@ -103,11 +103,10 @@ private static AppAccessToken buildAppAccessToken(Map<String, Object> response)
     public static AppAccessData requestAppId(AppAccessToken request) throws PlenigoException {
         Map<String, String> body = new LinkedHashMap<String, String>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java [hunk 4]
```diff
@@ -124,11 +123,11 @@ public static boolean hasUserBought(ProductAccessRequest request) throws Plenigo
         Map<String, Object> params = new LinkedHashMap<String, Object>();
         boolean hasAccess = true;
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java (hunk 2 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Yes, it tells me its  required

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java (hunk 3 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Yes, it tells me its required

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java [hunk 5]
```diff
@@ -151,11 +150,11 @@ public static boolean hasUserBought(ProductAccessRequest request) throws Plenigo
     public static void deleteCustomerApp(DeleteAppIdRequest request) throws PlenigoException {
         Map<String, Object> params = new LinkedHashMap<String, Object>();
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/MobileService.java [hunk 6]
```diff
@@ -44,11 +45,10 @@ private MobileService() {
     public static String verifyMobileSecret(String email, String mobileSecret) throws PlenigoException {
         Map<String, Object> body = new LinkedHashMap<String, Object>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/MobileService.java [hunk 7]
```diff
@@ -64,9 +64,8 @@ public static String verifyMobileSecret(String email, String mobileSecret) throw
     public static MobileSecretInfo getMobileSecret(String customerId) throws PlenigoException {
         Map<String, Object> body = new LinkedHashMap<String, Object>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/MobileService.java [hunk 8]
```diff
@@ -89,10 +88,9 @@ public static MobileSecretInfo createMobileSecret(String customerId, int mobileS
         }
         Map<String, String> body = new LinkedHashMap<String, String>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java (hunk 1 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/MobileService.java [hunk 9]
```diff
@@ -121,9 +119,8 @@ private static MobileSecretInfo buildMobileSecretInfo(Map<String, Object> respon
     public static boolean deleteMobileSecret(String customerId) throws PlenigoException {
         Map<String, Object> body = new LinkedHashMap<String, Object>();
         body.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java (hunk 4 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/ProductService.java [hunk 10]
```diff
@@ -68,9 +69,9 @@ public static ProductData getProductData(String productId) throws PlenigoExcepti
                 , new Object[]{productId, PlenigoManager.get().getCompanyId()});
         Map<String, Object> params = new HashMap<String, Object>();
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/ProductService.java [hunk 11]
```diff
@@ -89,13 +90,13 @@ public static ProductData getProductData(String productId) throws PlenigoExcepti
      */
     public static PagedList<ProductInfo> getProductList(int pageSize, String lastId) throws PlenigoException {
         Map<String, Object> params = new HashMap<String, Object>();
-        params.put(ApiParams.SECRET, PlenigoManager.get().getSecret());
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/AppManagementService.java (hunk 5 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/AppManagementService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/ProductService.java [hunk 12]
```diff
@@ -112,13 +113,13 @@ public static ProductData getProductData(String productId) throws PlenigoExcepti
      */
     public static PagedList<CategoryInfo> getCategoryList(int pageSize, String lastId) throws PlenigoException {
         Map<String, Object> params = new HashMap<String, Object>();
-        params.put(ApiParams.SECRET, PlenigoManager.get().getSecret());
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/ProductService.java [hunk 13]
```diff
@@ -379,9 +380,9 @@ public static CategoryData getCategoryData(String categoryId) throws PlenigoExce
                 , new Object[]{categoryId, PlenigoManager.get().getCompanyId()});
         Map<String, Object> params = new HashMap<String, Object>();
         params.put(ApiParams.COMPANY_ID, PlenigoManager.get().getCompanyId());
```
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Is the companyId needed here anymore?

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 6 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 7 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 8 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 9 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/ProductService.java (hunk 10 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/ProductService.java (hunk 11 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Fixed

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 6 above)
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Realy fixed?

File: src/main/java/com/plenigo/sdk/services/MobileService.java (hunk 7 above)
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/MobileService.java]: Realy fixed?

File: src/main/java/com/plenigo/sdk/services/ProductService.java (hunk 10 above)
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Realy fixed?

File: src/main/java/com/plenigo/sdk/services/ProductService.java (hunk 13 above)
Comment (robertholfeld) [inline src/main/java/com/plenigo/sdk/services/ProductService.java]: Not fixed.

**Pull request merged**